src/data_switch.rs

```diff
@@ -82,17 +82,26 @@ pub struct GeoPoint {
 /// represented by its vertices as a sequence of lat-lon points
 pub type Polygon = [GeoPoint];
 
-/// Container of series data
-#[derive(Debug, Clone, PartialEq)]
-pub struct SeriesCache {
+/// Data container
+///
+/// a [`new`](DataCache::new) method is provided to
+/// avoid the need to construct an R*-tree manually
+// TODO: check why we needed to derive PartialEq for SeriesCache
```

Comment:
Did you get to the bottom of this? Given that nothing broke, and it wasn't derived on SpatialCache, I'm inclined to think it was just an eager derivation. Feel free to remove the TODO